src/utils/operations/getDenominatedValue.ts

```diff
@@ -0,0 +1,22 @@
+import { DenominationConfig } from 'components/TransactionsInterpreter/TransactionsInterpreter';
+import { defaultNetwork } from 'reduxStore/slices';
+import { ServerTransactionType } from 'types/serverTransactions.types';
+import { denominate } from 'utils/operations/denominate';
+
+export function getDenominatedValue(
```

Comment:
deprecate and create new function named getFormattedAmount. Or if it was not present in dapp-core 1, directly change the name